Tools/TestWebKitAPI/Tests/WebKitCocoa/HSTS.mm

```diff
@@ -168,7 +168,12 @@ HTTPServer httpServer({
     EXPECT_EQ(httpServer.totalRequests(), 1u);
 }
 
+// FIXME when rdar://108167361 is resolved
+#if PLATFORM(MAC) && !defined(NDEBUG)
```

Comment:
If you want to disable this just for release, this should be 
`if PLATFORM(MAC) && defined(NDEBUG)`

Reply:
Done!